### ISSUE TITLE
Partners page needs UI consistency

### ISSUE THREAD

**Issue by wlcobb** [uno-isqa-8950/uno-cpi#1661]
Partners page from drop down
The bottom links are not consistent in sizing and one doesn't have an underline
Please remove the people icon in the buttons
